en-US/Players_Guide/04_Adventuring/Combat.md

```diff
@@ -0,0 +1,24 @@
+### Combat
+
+A typical combat encounter is a clash between two sides, a flurry of weapon swings, feints, parries, footwork, and spellcasting.
+The game organizes the chaos of combat into a cycle of rounds and turns.
+A **round** represents about 6 seconds in the game world.
+During a round, each participant in a battle takes a **turn**.
+The order of turns is determined at the beginning of a combat encounter, when everyone rolls initiative.
+Once everyone has taken a turn, the fight continues to the next round if neither side has defeated the other.
+
+[**General Terms**](./Combat_General_Terms.md)
+
+[**Movement and Position**](./Combat_Movement_and_Position.md)
+
+[**Actions in Combat**](./Combat_Actions.md)
+
+[**Making an Attack**](./Combat_Making_an_Attack.md)
+
+[**Cover**](./Combat_Cover.md)
+
+[**Damage and Healing**](./Combat_Damage_and_Healing.md)
+
+[**Mounted Combat**](./Combat_Mounted_Combat.md)
+
+[**Underwater Combat**](./Combat_Underwater_Combat.md)
```

Comment:
This section was missing from the Google Doc, so I added the WotC SRD version.
If it should be changed, we can obviously do that.